h2/src/main/org/h2/mvstore/rtree/BasicSpatialImpl.java

```diff
@@ -0,0 +1,74 @@
+/*
+ * Copyright 2004-2020 H2 Group. Multiple-Licensed under the MPL 2.0,
+ * and the EPL 1.0 (https://h2database.com/html/license.html).
+ * Initial Developer: H2 Group
+ */
+package org.h2.mvstore.rtree;
+
+import java.util.Arrays;
+
+/**
+ * Class BasicSpatialImpl.
+ *
+ * @author <a href='mailto:andrei.tokar@gmail.com'>Andrei Tokar</a>
+ */
+public final class BasicSpatialImpl implements Spatial
```

Comment:
I guess this class is a part of public API of standalone MVStore? Maybe we can find a better name for it? `DefaultSpatial`, `SimpleSpatial`, something else?

I think it would be nice to have this separate implementation, because users of standalone MVStore should not see methods of our internal `Value` class in `SpatialKey`.

Reply:
It's not in public API, this class does not have to be public. Better name - sure.